roles/confluent.kafka_connect/templates/connect-distributed.properties.j2

```diff
@@ -1,159 +1,4 @@
-# Maintained by Ansible
-
-{% for key, value in kafka_connect.properties.items() %}
+# Written by Ansible
+{% for key, value in kafka_connect_final_properties|dictsort %}
 {{key}}={{value}}
 {% endfor %}
-
-# Kafka Connect Configuration
-listeners={{kafka_connect_http_protocol}}://0.0.0.0:{{kafka_connect_rest_port}}
```

Comment:
we should lave a comment on where the properties are merged together to generate the final list that the remaining codes prints out.